precompiles/ArbOwnerPublic.go

```diff
@@ -28,3 +29,11 @@ func (con ArbOwnerPublic) IsChainOwner(c ctx, evm mech, addr addr) (bool, error)
 func (con ArbOwnerPublic) GetNetworkFeeAccount(c ctx, evm mech) (addr, error) {
 	return c.State.NetworkFeeAccount()
 }
+
+// Gets the infrastructure fee collector
+func (con ArbOwnerPublic) GetInfraFeeAccount(c ctx, evm mech) (addr, error) {
+	if c.State.FormatVersion() < 5 {
+		return common.Address{}, errors.New("")
```

Comment:
I believe we have a method of versioning precompiles that doesn't change gas accounting as this does, cc @RachelFranks could you advise on how to version them?

Reply:
When configuring precompiles after calling `MakePrecompile()` in `precompiles.go`, you can specify an ArbOS version requirement by setting `arbosVersion` to a non-zero value